beswifty.com.website.json

```diff
@@ -0,0 +1,24 @@
+{
+   "providerId":"beswifty.com",
+   "providerName":"Beswifty",
+   "serviceId":"website",
+   "serviceName":"Swifty Site",
+   "version": 4,
+   "logoUrl":"https://beswifty.com/wp-content/uploads/2019/12/Swifty-Logo_Tagline.svg",
+   "description":"Enables a domain to work with Swifty Sites",
+   "syncRedirectDomain":"*.beswifty.com",
+   "records":[
+      {
+         "type":"CNAME",
+         "host":"www",
+         "pointsTo":"@",
+         "ttl":600
+      },
+      {
+         "type":"A",
+         "host":"@",
+         "pointsTo":"%host%",
```

Comment:
`%host%` is a reserved variable equal to `host` parameter of `/apply` call.
Also, if the IP address is variable, the template shall either use signed apply (specify syncPubKeyDomain) or be marked as insecure (warnPhishing set to true).
See https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#423-security-considerations for details

Reply:
This version is mainly for the internal use. So we are okay setting warnPhishing to true in this version